packages/cli/lib/sandbox-create.js

```diff
@@ -2,10 +2,7 @@ const SpinniesManager = require('./SpinniesManager');
 const {
```

Comment:
Nit, but most of the files in the CLI use camel case. Would you mind updating this for consistency?

Reply:
yes not a prob, I did also notice inconsistencies but I'll at least get these to camel case